pkg/utils/annotation.go

```diff
@@ -0,0 +1,10 @@
+package utils
+
+const (
+	// DeployModeAnnotationKey describe the klusterlet deploy mode when importing a managed cluster.
+	DeployModeAnnotationKey string = "import.open-cluster-management.io/klusterlet-deploy-mode"
```

Comment:
do we still need this annotation?

Reply:
If you are not using it either, I think we are safe to delete it.

Reply:
Removed.